Lib/ufo2ft/__init__.py

```diff
@@ -88,7 +88,7 @@ def _getMtiFeatures(ufo, mtiFeaFiles=None):
         # For now, the side input wins over any MTI features inside the UFO.
         features = {}
         for table, feapath in mtiFeaFiles.items():
-            with open(feapath, "r") as feafile:
+            with open(feapath, "rb") as feafile:
```

Comment:
are you sure mtiLib wants bytes? It seems to me it opens the files as text and decodes them as UTF-8:
https://github.com/fonttools/fonttools/blob/4e9806a/Lib/fontTools/mtiLib/__init__.py#L1178